src/Issue/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { default as PT } from 'prop-types'
+import Icon from '../Icon'
+
+const IssueIcon = ({ variant }) =>
+  ({
+    success: <Icon name="success" className="ola_issue-status" />,
+    warning: <Icon name="warning" className="ola_issue-status" />,
+    suggested: <Icon name="help" className="ola_issue-status" />,
+  }[variant] || <Icon name="error" className="ola_issue-status" />)
+
+const IssueTitle = ({ title, link }) => (
+  <>
+    <span className="ola_issue-title">{title}</span>
+    {link && <Icon name="chevronRight" />}
+  </>
+)
+
+const Issue = ({ title, variant, link }) => (
```

Comment:
I miss a `size` property with two possible values: `small` and `medium`:
- `small`: The title is `callout` (and bold)
- `medium`: The title is `body` (and bold).

Reply:
done!